### PR TITLE
changed names

### PR DESCRIPTION
**Pull request by melvinshaji22**

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -68,7 +65,7 @@ this.router.navigate(['description/',id])
 }
 favFunc(data:any){
   this.service.updateFav(data)
-  .subscribe(d=>{console.log('-----',d);
+  .subscribe(d=>{
```
Comment (Sushmitha-Vk) [inline src/app/dashboard/dashboard.component.ts]: why is d declared..?  its not used at all inside subscribe

File: src/app/membership/membership.component.ts [hunk 2]
```diff
@@ -62,7 +59,7 @@ export class MembershipComponent implements OnInit {
   } 
   form4(){
     this.service.createELEMENT_DATA({...this.firstFormGroup.value})
-    .subscribe(a=>{console.log("-----a--",a);
+    .subscribe(data=>{
```
Comment (Sushmitha-Vk) [inline src/app/membership/membership.component.ts]: data not used right dont declare it at all

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/add-book/add-book.component.ts [hunk 3]
```diff
@@ -30,23 +30,18 @@ export class AddBookComponent implements OnInit {
     
   }
 submit(data:any){
-  this.serv.createELEMENT_DATA({...data.value,bookName:data.value['bookName']}).subscribe(s =>{
-   window.location.reload();
-    
-    
-  });
+  this.serv.createELEMENT_DATA({...data.value,bookName:data.value['bookName']}).subscribe()
+  window.location.reload();
```
Comment (Sushmitha-Vk) [inline src/app/add-book/add-book.component.ts]: this is wrong, `window.location.reload(); ` should be inside subscribe only. I just said to remove the value declaration

File: src/app/add-book/add-book.component.ts [hunk 4]
```diff
@@ -30,23 +30,18 @@ export class AddBookComponent implements OnInit {
...
 }
 update(data:any){
   
-this.serv.editELEMENT_DATA({...data, id: this.data.id}).subscribe(u=>{
-  
-  this.dialog.closeAll();
-   window.location.reload();
-
-})}
+this.serv.editELEMENT_DATA({...data, id: this.data.id}).subscribe()
```
Comment (Sushmitha-Vk) [inline src/app/add-book/add-book.component.ts]: same as above

File: src/app/dashboard/dashboard.component.ts [hunk 5]
```diff
@@ -65,8 +65,8 @@ this.router.navigate(['description/',id])
 }
 favFunc(data:any){
   this.service.updateFav(data)
-  .subscribe(d=>{
+  .subscribe()
   window.location.reload()
-  })
```
Comment (Sushmitha-Vk) [inline src/app/dashboard/dashboard.component.ts:73]: same as above

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/dashboard/dashboard.component.ts (hunk 1 above)
Comment (melvinshaji22) [reply src/app/dashboard/dashboard.component.ts]: resolved

**Review by Sushmitha-Vk — APPROVED** [review]

**Pull request merged**